runtime/vm/classsupport.c

```diff
@@ -307,11 +307,19 @@ internalFindClassString(J9VMThread* currentThread, j9object_t moduleName, j9obje
 		char localBuf[J9VM_PACKAGE_NAME_BUFFER_LENGTH];
 		char *utf8Name = NULL;
 		UDATA utf8Length = 0;
+		UDATA stringFlags = ((CLASSNAME_INVALID != allowedBitsForClassName) ? J9_STR_NULL_TERMINATE_RESULT : (J9_STR_NULL_TERMINATE_RESULT | J9_STR_XLAT));
```

Comment:
Same comment as above - assignment and if.

Reply:
Updated the code against the suggestion above.